chatcommands.py

```diff
@@ -719,28 +719,45 @@ def errorlogs(count):
     return fetch_lines_from_error_log(count or 2)
 
 
-@command(whole_msg=True, aliases=["ms-status", "ms-down", "ms-up"], give_name=True)
+@command(whole_msg=True, aliases=["ms-status", "ms-down", "ms-up", "ms-down-force", "ms-up-force"], give_name=True)
 def metasmoke(msg, alias_used):
     if alias_used in {"metasmoke", "ms-status"}:
         status_text = [
-            "metasmoke is up. Current failure count: {} ({id})".format(GlobalVars.metasmoke_failures,
+            "metasmoke is up. Current failure count: {} ({id})".format(GlobalVars.MSStatus.get_failure_count(),
                                                                        id=GlobalVars.location),
-            "metasmoke is down. Current failure count: {} ({id})".format(GlobalVars.metasmoke_failures,
+            "metasmoke is down. Current failure count: {} ({id})".format(GlobalVars.MSStatus.get_failure_count(),
                                                                          id=GlobalVars.location),
         ]
-        return status_text[GlobalVars.metasmoke_down]
+        if GlobalVars.MSStatus.is_up:
+            # True = 1 and False = 0 is a legacy feature
+            # Better not to use them
+            return status_text[0]
+        else:
+            return status_text[1]
     # The next aliases/functionalities require privilege
     if not is_privileged(msg.owner, msg.room):
         raise CmdException(GlobalVars.not_privileged_warning)
 
     if alias_used == "ms-down":
-        GlobalVars.metasmoke_down = True
-        GlobalVars.metasmoke_failures = 999
-        return "metasmoke is now considered down."
+        Metasmoke.AutoSwitch.switch_auto(True)
```

Comment:
There's a lot of redundant code and text in this next section. It seems that it would be possible to consolidate this all down into a few statements like:

```
is_force = "force" in alias_used
going_up = 'up' in alias_used
Metasmoke.AutoSwitch.switch_auto(is_force)
if going_up:
    Metasmoke.ms_up()
else:
    Metasmoke.ms_down()
return "Successfully set metasmoke status to **{}**.".format("up" if going_up else "down") +\
       " Automatic metasmoke status switch is currently **{}abled**.".format("dis" if is_force else "en")
```